pkg/services/kmsproviders/kmsproviders.go

```diff
@@ -0,0 +1,11 @@
+package kmsproviders
+
+import "github.com/grafana/grafana/pkg/services/secrets"
+
+const (
+	Default = "secretKey"
+)
+
+type Service interface {
+	Provide() (map[string]secrets.Provider, error)
```

Comment:
It returns a list of providers, so maybe is better to use `GetProviders()` instead?

Reply:
Well, this service is typically responsible for initializing providers (based on settings) and "provide" them. So, I'm not sure if `GetProviders()` or simply `Providers()` would make more sense. It's not a regular getter method, that's precisely why I tried to use a different name other than the common `GetX()`.

In fact, if at some point we add support for "reloads", every call to this method would re-read settings and provide different providers based on the latest updates.

What do you think? cc/ @undef1nd

Reply:
Thanks for the comment, Selene!

If nobody minds, I'd leave as is because I don't have a better name right now.